cityofthebigshoulders.js

```diff
@@ -4736,6 +4736,14 @@ function (dojo, declare) {
             this.placeDemand(demand);
             this.placeOnObject(demand.card_type, 'main_board');
             this.slideToObject(demand.card_type, demand.card_location).play();
+
+            // Update counter localy
+            if (this.gamedatas.counters['demand_deck_count'])
```

Comment:
I don't think this is needed since the counter always exists (it's create during GetAllDatas as soon as the game is created)

Reply:
True, I tested a full decade without it and all fine